soh/src/code/z_parameter.c

```diff
@@ -746,12 +746,19 @@ void func_80083108(GlobalContext* globalCtx) {
             }
         } else if (msgCtx->msgMode == MSGMODE_NONE) {
             if ((func_8008F2F8(globalCtx) >= 2) && (func_8008F2F8(globalCtx) < 5)) {
-                if (gSaveContext.buttonStatus[0] != BTN_DISABLED) {
-                    sp28 = 1;
-                }
+                if (func_8008F2F8(globalCtx) == 2) {
```

Comment:
Shouldn't there be a cvar check here?

Reply:
The cvar check is made later on in that block.